README.md

```diff
@@ -461,10 +461,13 @@ otherwise lead to nasty loops.
 `wayland-sessions`. At this point one can cancel and continue to the normal
 login shell.
 
+`systemd-cat -t uwsm-start […]` runs the command given to it with its stdout
```

Comment:
and stderr

Reply:
"...useful options may be added  like `--stderr-priority=err` if desired." or something like that.